api/backend.go

```diff
@@ -266,7 +267,17 @@ func CredentialsTokenHandler(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(responseBody)
 }
 
+func getBinaries() {
+	cmd := exec.Command("python3", "get_binaries.py")
```

Comment:
I think it's practice to run `./get_binaries.py` and add the `python3` shebang to the top of the file since this is treated as an executable.

Reply:
I think the only way to do this is to add a shebang to the python file. Unfortunately, that only works for Unix/MacOS machines. Windows you have to download something to make it an executable.